### DISCUSSION TITLE
Section 02: A question related to the input of loss functions

### DISCUSSION THREAD

**Discussion by RishavB777** [mrdbourke/pytorch-deep-learning#344] [Q&A]
```
for epoch in range(epochs):
    ### TRAINING
    model_4.train()

    y_logits = model_4(X_blob_train).squeeze()
    y_pred = torch.argmax(torch.softmax(y_logits, dim=1), dim=1)

    loss = loss_fn(y_logits, y_blob_train)
```

Why does the cross entropy loss take raw logits instead of labels while comparing with y_blob_train which has data in the form of labels(0,1,2,3)?
